pgcli/pgbuffer.py

```diff
@@ -4,13 +4,20 @@
 
 
 class PGBuffer(Buffer):
-    def __init__(self, always_multiline, *args, **kwargs):
+    def __init__(self, always_multiline, multiline_mode, *args, **kwargs):
         self.always_multiline = always_multiline
+        self.multiline_mode = multiline_mode
 
         @Condition
         def is_multiline():
             doc = self.document
-            return self.always_multiline and not _multiline_exception(doc.text)
+            if not always_multiline:
```

Comment:
This has to be `if not self.always_multiline`. That makes sure we're checking the instance variable instead of the local variable. We modify the instance variable when the user presses F3 to toggle the multi-line.